src/templates/_common/components/forms.html.tt

```diff
@@ -59,7 +59,8 @@
         [% END -%]
     [% END %]
         [% UNLESS no_error %]
-        <p id="[% msg_id || 'msg_form' %]" class="error-msg no-margin invisible"></p>
+        <p id="[% msg_id || 'msg_form' %]" class="[% is_centered ? 'error-msg no-margin center-text invisible' : 'error-msg no-margin
+invisible' %]"></p>
```

Comment:
1. why is there a line break?
2. better written as the following to avoid repeating classes:

`<p id="[% msg_id || 'msg_form' %]" class="error-msg no-margin invisible[% is_centered ? ' center-text' : '' %] "></p>`

Reply:
@negarn , noted with thanks.